app/scripts/lib/app-start.js

```diff
@@ -641,27 +641,6 @@ define(function (require, exports, module) {
       }
     },
 
-    _getErrorPage: function (err) {
-      if (AuthErrors.is(err, 'INVALID_PARAMETER') ||
```

Comment:
I'm reverting back to the list, ultimately I'm not happy with how this PR came out.